src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+$cardTitleColor: #34568b;
+$transitionDuration: 0.3s;
+$navigationLinkColor: #00acdc;
```

Comment:
we usually use more generic names for color variables. If you will need the same color at some point, but not in the context of the card title color - will this name be a good choice? That's why we usually have it like `$primary-green`, `$secondary-green`, etc.